index.js

```diff
@@ -454,6 +454,12 @@ const runTestsUsingCommandLine = async () => {
     cmd.push(envInput)
   }
 
+  const quiet = core.getInput('quiet')
+  if (quiet) {
+    cmd.push('--quiet')
+    cmd.push(quiet)
```

Comment:
since this is a boolean flag, you should only pass it if true

```
if (quiet) {
  cmd.push('--quiet')
}
```

Reply:
changes applied